### PR TITLE
edit ratu_serializers

### PR DESCRIPTION
**Pull request by Baranovsky1986**
modify ratu_serializers using ModelSerializer

### CODE REVIEW

File: data_ocean/serializers/ratu_serializers.py [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
-    region = serializers.CharField(max_length=30)
-    district = serializers.CharField(max_length=100)
-    city = serializers.CharField(max_length=100)
-    name = serializers.CharField(max_length=100)
-    koatuu = serializers.CharField(max_length=10)
+class CitydistrictSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Citydistrict
+        fields = ('id', 'region', 'district', 'city', 'name', 'koatuu')
+    serializers.CharField(max_length=10)
```
Comment (criticus) [inline data_ocean/serializers/ratu_serializers.py:24]: what is this line for?

Comment (Baranovsky1986) [reply data_ocean/serializers/ratu_serializers.py]: i fixed it

File: data_ocean/serializers/ratu_serializers.py [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
 
-class RegionSerializer(serializers.Serializer):
-    name = serializers.CharField(max_length=30)
-    koatuu = serializers.CharField(max_length=10)
-    
-class DistrictSerializer(serializers.Serializer):
+class RegionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Region
+        fields = '__all__'
```
Comment (criticus) [inline data_ocean/serializers/ratu_serializers.py:7]: we definitely don't want to display all fields. For example, we never want to show the field "deleted"

Comment (Baranovsky1986) [reply data_ocean/serializers/ratu_serializers.py]: now we show all fields without time - fields

**Comment by Baranovsky1986** [comment]
unnecessary

**Pull request closed without merging**